### PR TITLE
Create custom `Location` struct 

### PR DESCRIPTION
**Pull request by lukasz-szyszkowski**
The idea behind these changes is to create custom `Location` object to be independent of `CoreLocation` objects

### CODE REVIEW

File: Sources/AblyAssetTrackingCore/Model/Location.swift [hunk 1]
```diff
@@ -0,0 +1,100 @@
+
+import Foundation
+
+public struct Location: Equatable {
+    
+    public let coordinate: LocationCoordinate
```
Comment (KacperKluka) [inline Sources/AblyAssetTrackingCore/Model/Location.swift:6]: Every other field has a comment but this one doesn't, please add a comment for this field :wink:

**Review by KacperKluka — CHANGES_REQUESTED** [review]
Just one small thing :wink:

File: Sources/AblyAssetTrackingCore/Model/Location.swift (hunk 1 above)
Comment (lukasz-szyszkowski) [reply Sources/AblyAssetTrackingCore/Model/Location.swift:9]: > Every other field has a comment but this one doesn't, please add a comment for this field 😉

added here e6d96ed5ae10cb04c958b27897912a546674995a

**Review by KacperKluka — APPROVED** [review]
LGTM :+1:

**Review by maratal — APPROVED** [review]

**Pull request merged**